src/vm/object_model.rs

```diff
@@ -48,17 +48,21 @@ pub trait ObjectModel<VM: VMBinding> {
     //
     // MMTk reserved PolicySpecific side metadata offsets:
     //
-    //  1 - MarkSweep Alloc bit:
+    //  1 - MarkSweep Chunk Mark byte:
     //      - Offset `0x0` on 32-bits
```

Comment:
This is global, right?

Reply:
Indeed, if bindings use `.._SIDE_METADATA_VM_BASE_ADDRESS` instead of `.._SIDE_METADATA_BASE_ADDRESS`, they don't have to know these offsets.
So, we'd better move these two lists to where `.._SIDE_METADATA_VM_BASE_ADDRESS` are defined (e.g. `src/util/metadata/side_metadata/constants.rs`)

Reply:
ignore this comments, as it's been fixed in Yi's next PR

Reply:
Ah sorry -- I didn;t update the comments to reflect that.